core-bundle/src/Resources/contao/modules/ModuleChangePassword.php

```diff
@@ -169,6 +170,19 @@ protected function compile()
 			}
 
 			$strFields .= $objWidget->parse();
+			$arrWidgets[] = $objWidget;
```

Comment:
how about we only collect the "finalizable" widgets to not need to loop over them again (applies to the form code as well)?